package.json

```diff
@@ -35,7 +35,7 @@
     "clean:all": "npm run analyze:clean && npm run test:clean && npm run build:clean",
     "generate": "plop --plopfile internals/generators/index.js",
     "lint": "npm run lint:js",
-    "lint:eslint": "eslint --ignore-path .gitignore --ignore-pattern internals/scripts",
+    "lint:eslint": "eslint --ignore-path .gitignore --ignore-pattern internals/scripts --fix",
```

Comment:
If a staged file is linted before commiting it is fixed, but the changes are not staged and the commit is made. This way you can commit incorrectly formatted files. `lint-staged` should not use the `--fix` flag.

Reply:
Agreed! There's an alternative way to fix it though (which I've been using for a long time):

![image](https://user-images.githubusercontent.com/8948127/41240912-481a4cc8-6da4-11e8-90fe-d431ea8d8850.png)

Reply:
(This will add the fixes to your commit.)

Reply:
@gretzky What do you think of this? Remove the --fix flag or keep it but make sure they're added to the commit?